doc/man/man3/crypto_lock.3monocypher

```diff
@@ -226,18 +226,36 @@ attacker's interference.
 .Fa plain_text
 does not need to be wiped if the decryption fails.
 .Sh EXAMPLES
+The following examples assume that a function called
+.Fn random_bytes
+exists.
+It fills the given buffer with cryptographically secure random
+bytes
+(see
+.Xr intro 3monocypher
+for some advice on how to accomplish this yourself).
+The function has this prototype:
+.Ft void
+.Fo random_bytes
+.Fa "uint8_t *buf"
+.Fa "size_t len"
+.Fc
+.Pp
 Encryption:
 .Bd -literal -offset indent
 uint8_t       key        [32];  /* Random, secret session key  */
 const uint8_t nonce      [24];  /* Use only once per key       */
 uint8_t       plain_text [500]; /* Secret message              */
 uint8_t       mac        [16];  /* Message authentication code */
 uint8_t       cipher_text[500]; /* Encrypted message           */
+random_bytes(key, 32);
```

Comment:
So, this one could be replaced by one of the following:

    random_bytes(key, 32) /* use the OS to implement this function */
    arc4random_buf(key, 32);

If we're starting to use standard UNIX functions, I believe I prefer arc4random_buf(3), which requires less extraneous explanation (possibly an `#include` line, no more). Plus, it will follow our own advice and use the damn OS to get random stuff.